logfire/_internal/db_statement_summary.py

```
@@ -79,7 +79,10 @@ def summarize_query(db_statement: str) -> str | None:
         return select(expr, table, match_end=select_match.end(), db_statement=db_statement, ctes=ctes)
     elif insert_match := INSERT_RE.match(db_statement):
         table, columns, values = insert_match.groups()
-        return f'INSERT INTO {table} {truncate(columns, 25)} VALUES {truncate(values, 25)}'
+        return truncate(
+            f'INSERT INTO {table} {truncate(columns, 25)} VALUES {truncate(values, 25)}',
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)